### PR TITLE
no POST for auxiliary resources

### PR DESCRIPTION
**Pull request by bourgeoa**
issue #1528 
Passes all tests locally
tests reviewed and added

### CODE REVIEW

File: lib/ldp.js [hunk 1]
```diff
@@ -147,7 +147,7 @@ class LDP {
     debug.handlers('POST -- On parent: ' + containerPath)
     // prepare slug
     if (slug) {
-      if (this.isAuxResource(slug, extension)) throw error(403, 'POST is not allowed for auxiliary resources')
+      if (this.isAuxResource(slug, extension)) throw error(403, 'POST is not allowed for auxiliari resources')
```
Comment (TallTed) [inline lib/ldp.js]:
```suggestion
      if (this.isAuxResource(slug, extension)) throw error(403, 'POST is not allowed for auxiliary resources')
```
Why the change from correct spelling to incorrect?

Comment (bourgeoa) [reply lib/ldp.js]: @TallTed just a fake commit that is reverse to go to the tests that are now on github actions
